python-frontend/src/main/java/org/sonar/python/semantic/v2/types/PropagationVisitor.java

```diff
@@ -75,6 +76,16 @@ public void visitFunctionDef(FunctionDef functionDef) {
     propagationsByLhs.computeIfAbsent(symbol, s -> new HashSet<>()).add(definition);
   }
 
+  @Override
+  public void visitClassDef(ClassDef classDef) {
+    super.visitClassDef(classDef);
+    Name name = classDef.name();
+    var symbol = name.symbolV2();
+    Definition definition = new Definition(symbol, name);
```

Comment:
super minor, I think from the code style perspective it will look better if all 3 variables there will be defined with `var` of with a type predicate, to keep similarity